### PR TITLE
Add JSON Schema Conference blog article

### PR DESCRIPTION
**Pull request by valeriahhdez**
**What kind of change does this PR introduce?**
This PR adds an article to the blog recapping the JSON Schema conference hosted during the APIdays Paris 2024. 

**Issue Number:** No issue

-  Closes #___ 
-  Related to #___ 
-  Others? 

**Screenshots/videos:**

**If relevant, did you update the documentation?**

No

**Summary**

JSON Schema hosted its first conference track during the APIdays Paris Conference 2024. This article aims to update the community on success of the event and what we learned from it. 

**Does this PR introduce a breaking change?** No

### CODE REVIEW

File: pages/blog/posts/apidays-paris-2024-recap.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+tags:
+  - News
+type: Community
+cover: /img/posts/2024/apidays-paris-2024-recap/json-schema-conference.png
+authors:
+  - name: Valeria Hernandez
+    photo: /img/avatars/valeria.webp
+    twitter: valeriahhdez
+    byline: Technical writer
+excerpt: "API Days Paris 2023 has been an amazing experience for JSON Schema"
```
Comment (Relequestual) [inline pages/blog/posts/apidays-paris-2024-recap.md:13]:
```suggestion
excerpt: "API Days Paris 2024 has been an amazing experience for JSON Schema"
```

Comment (Relequestual) [reply pages/blog/posts/apidays-paris-2024-recap.md:13]: Force of habbit? =D
After this is fixed, will approve.

**Review by gregsdennis — APPROVED** [review]
Really good article!  Just a couple nitpicks.

File: pages/blog/posts/apidays-paris-2024-recap.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+## The first JSON Schema conference
+
+The inaugural [JSON Schema conference track](https://conference.json-schema.org/) at APIdays Paris 2024 was a resounding success. As part of the larger conference theme "The Future API Stack for Mass Innovation," the track showcased JSON Schema's crucial role in modern API development and data contract design. 
+
+<div className='flex flex-wrap justify-center items-center gap-4 w-full'>
+    <img className='max-w-400 px-20' src='/img/posts/2024/apidays-paris-2024-recap/json-schema-team.jpg' alt="Five people standing at a booth labeled 'Standards'. They are smiling and looking at the camera. From left to right, their names are Benja Granados, Thomas Clark, Andreas Eberhart, Juan Cruz Viotti, and Ben Hutton."/>
+    <p>Community members at the standards booth shared with AsyncAPI, OpenAPI, and GraphQL. From left to right: Benja Granados (community manager), Thomas Clark (Octue CEO), Andreas Eberhart (JSON Schema ambassador), Juan Cruz Viotti (TSC member), and Ben Hutton (TSC member).</p>
+</div>
```
[the remark this answers is not in the document]
Comment (valeriahhdez) [reply pages/blog/posts/apidays-paris-2024-recap.md]: Not that I'm aware of. To improve accessibility, I've switched from <div> and <p> to the more semantically correct <figure> and <figcaption> elements for the image and caption.

File: pages/blog/posts/apidays-paris-2024-recap.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+This year's agenda highlighted the increasing role of AI in API development, exploring how AI can be used to automate tasks, improve API design, and enhance API performance.
+Continuing our successful partnership with [AsyncAPI](https://www.asyncapi.com/en) and [OpenAPI](https://www.openapis.org/), this year's event marked a significant milestone with the inaugural JSON Schema conference track. This dedicated track, sponsored by [Octue](https://www.octue.com/), delved deep into the world of JSON Schema, exploring its role in API design, validation, and documentation.
+
+## The first JSON Schema conference
+
+The inaugural [JSON Schema conference track](https://conference.json-schema.org/) at APIdays Paris 2024 was a resounding success. As part of the larger conference theme "The Future API Stack for Mass Innovation," the track showcased JSON Schema's crucial role in modern API development and data contract design.
```
Comment (gregsdennis) [inline pages/blog/posts/apidays-paris-2024-recap.md:23]: "Apidays"

Sorry. I thought that you'd search for all of them.

**Review by benjagm — APPROVED** [review]
Great work Valeria!!

**Comment by DhairyaMajmudar** [comment]
Awsm PR @valeriahhdez, pls. merge the latest changes with the main to your branch to fix the failing workflow.

**Pull request merged**